src/main/scala/com/microsoft/hyperspace/index/IndexManager.scala

```diff
@@ -104,4 +104,15 @@ trait IndexManager {
    * @return Index metadata and statistics as a [[DataFrame]].
    */
   def index(indexName: String): DataFrame
+
+  /**
+   * Get IndexLogEntry of the given index log version.
+   *
+   * @param indexName Name of the index to get.
+   * @param logVersion Index log version to retrieve.
+   * @return IndexLogEntry if the index of the given log version exists, otherwise None.
+   */
+  def getIndexLogEntry(
```

Comment:
We have `def getIndexes(states: Seq[String] = Seq()): Seq[IndexLogEntry]`. Should we make the naming consistent? Also, we should expose an API to return log versions: https://github.com/microsoft/hyperspace/pull/272#discussion_r598210846.

Reply:
If you want, you can create a separate PR that just updates `IndexManager` APIs, but I am also fine if you do it in this PR.

Reply:
will do this with a follow up PR

Reply:
sounds good!